app/src/main/java/org/fossasia/openevent/general/utils/Utils.kt

```diff
@@ -124,23 +124,6 @@ object Utils {
     fun requireDrawable(@NonNull context: Context, @DrawableRes resId: Int) = AppCompatResources
         .getDrawable(context, resId) ?: throw IllegalStateException("Drawable should not be null")
 
-    fun getAnimFade(): NavOptions {
-        val builder = NavOptions.Builder()
-        builder.setEnterAnim(R.anim.fade_in)
-        builder.setExitAnim(R.anim.fade_out)
-        builder.setPopEnterAnim(R.anim.fade_in)
-        builder.setPopExitAnim(R.anim.fade_out)
-        return builder.build()
-    }
-
-    fun getAnimSlide(): NavOptions {
-        val builder = NavOptions.Builder()
-        builder.setEnterAnim(R.anim.slide_in_right)
-        builder.setExitAnim(R.anim.slide_out_left)
-        builder.setPopEnterAnim(R.anim.slide_in_left)
-        builder.setPopExitAnim(R.anim.slide_out_right)
-        return builder.build()
-    }
```

Comment:
I don't think these should remove. It can be used later for any fragment navigation.